index.html

```diff
@@ -257,13 +286,13 @@ <h3 id="form-headline">
     <!-- Footer -->
     <section class="footer-container">
       <ul class="footer-button-group">
-        <li class="footer-social-media-button"><a href="https://github.com/mperezbazan"><img src="./images/GitHub-dark.svg" alt="Github icon"/></a></li>
+        <li class="footer-social-media-button"><a href="https://github.com/mperezbazan"><img src="./images/GitHubDark.svg" alt="Github icon"/></a></li>
```

Comment:
- [ ] Kindly make sure you use the correct path for your GitHub icon as required by the previous reviewer on my side it is still showing the broken link for the image, and the image is not displayed as expected. check the picture below of how it looks on my side 

## Your design 

![image](https://user-images.githubusercontent.com/95297251/193119588-55acbae1-8a12-461d-9f2d-8a761e45443b.png)

## Figma design 


![](https://user-images.githubusercontent.com/103154117/193107772-6ae86e43-3834-455e-a0b9-b72142d1822b.png)